lib/api/model/model.dart

```diff
@@ -275,6 +281,15 @@ sealed class Message {
   final String? matchContent;
   final String? matchSubject;
 
+  static Object? _readReactions(Map json, String key) {
+    final list = (json[key] as List<dynamic>);
+    return list.isNotEmpty ? list : null;
+  }
+
+  static Object _reactionsToJson(Reactions? value) {
+    return value ?? [];
```

Comment:
Ah I see, so this is what fixed those test failures (https://github.com/zulip/zulip-flutter/pull/286#pullrequestreview-1605820760).